Exercise6.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Exercise 6
+
+# In[1]:
+
+
+def calculation(a, b):
+    addition = a + b
+    subtraction = a - b
+    return addition, subtraction
+
+
+# In[2]:
+
+
+res = calculation(40, 10)
+print(res)
+
+
+# In[5]:
+
+
+def sort_words(hyphen_str):
+    txt = hyphen_str.split('-')
+    txt.sort()
+    print('-'.join(txt))
+
+sort_words('green-red-yellow-black-white')
+
+
+# In[15]:
+
+
+test_number = 8128
+
+def perfect_number():
```

Comment:
function is not taking argument input as supposed to, but reading form global variable. make sure to fix the input.